.azure-pipelines/weekly_tasks.yml

```diff
@@ -1,28 +0,0 @@
-trigger:
-  branches:
-    exclude:
-      - '*'
-
-pr: none
-
-schedules:
-  - cron: "0 0 * * 6"
-    displayName: 'Weekly Recurring Jobs'
-    branches:
-      include:
-        - 'main'
```

Comment:
Is this implied in the GH actions workflow?

Reply:
It works for the package build without. Not sure it was necessary on Azure.